packages/documentation/plugins/remark/gatsby-remark-mdx-mermaid/index.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
+const visit = require('unist-util-visit');
+const remark = require('remark');
+const mermaid = require('./remark-mermaid');
+const vfile = require('to-vfile');
+
+module.exports = ({ markdownAST, markdownNode }, { language = 'mermaid', theme = 'default' } = {}) => {
+  visit(markdownAST, 'code', async (node) => {
+    const lang = (node.lang || '').toLowerCase();
+    const fileContent = vfile.readSync(markdownNode.fileAbsolutePath);
+
+    if (lang === language) {
+      await remark()
+        .use(mermaid)
+        .process(fileContent, (err, file) => {
+          if (err) throw err;
+
+          const mermaidSrcMatch = file.messages.filter(
+            item => item.message.source === node.value
+          )[0];
+
+          node.type = 'html';
+          node.value = `<img src="${mermaidSrcMatch.message.imgSrc}" />`;
```

Comment:
I'm going to guess that this will break on the github.io site, since it won't have the site prefix in this path. I'm not sure how easy that is to get in here.